formats/developers/6d-7d-and-8d-storage.txt

```diff
@@ -206,29 +205,41 @@ mosaic as above or to overlap e.g.
 
     E   <Plane TheC="0" TheT="12" TheZ="0" PositionX="80" PositionY="110" PositionZ="0.1"/>
 
-Sample Files
-^^^^^^^^^^^^
+Sample files
+------------
 
+Sample files are available from the
+:image_downloads:`Modulo <OME-TIFF/2015-01/modulo/>` folder of the image downloads resource.
 
 SPIM
-""""
-
-:schema_plone:`SPIM-modulo-sample.ome.tiff <Samples/2015-01/SPIM-modulo-sample.ome.tiff>`
+^^^^
 
-A 2x2 tile of planes each recorded at 4 angles.
+- :image_downloads:`SPIM-ModuloAlongZ.ome.tiff <OME-TIFF/2015-01/modulo/SPIM-ModuloAlongZ.ome.tiff>` - a 2x2 tile of planes
+  each recorded at 4 angles.
 
 Big Lambda
-""""""""""
+^^^^^^^^^^
 
-:schema_plone:`LAMBDA-modulo-sample.ome.tiff <Samples/2015-01/LAMBDA-modulo-sample.ome.tiff>`
-
-Excitation of 5 wavelength [Λ, big-lambda] each recorded at 10 emission
-wavelength ranges [λ, lambda].
+- :image_downloads:`LAMBDA-ModuloAlongZ-ModuloAlongT.ome.tiff <OME-TIFF/2015-01/modulo/LAMBDA-ModuloAlongZ-ModuloAlongT.ome.tiff>` -
+  excitation of 5 wavelength [Λ, big-lambda] each recorded at 10 emission
```

Comment:
At least in the case of FLIM I try to use real as in "real-time" to distinguish it from the ModuloAlong dimension which isn;'t real-time but time relative to the excitation pulse.